analysis/supplementary-materials/Package_data_workflow.R

```diff
@@ -1,2 +1,5 @@
 # Create data-raw folder to compute package datasets ----------------------
 usethis::use_data_raw()
+
+# Create data folder with data to be made available as a part of the pacakge
+usethis::use_data(emu_theses, overwrite = TRUE)
```

Comment:
I renamed the object in the paper from `emu_theses` to `emu`. Should this be updated here and in the data preparation script as well?